src/routes/docs/products/ai/integrations/fal-ai/+page.markdoc

```diff
@@ -0,0 +1,216 @@
+        },
+      });
+
+      const json = await response.json();
+
+      if (!json.ok || json.error) {
+        alert(json.error);
+      }
+
+      return json;
+    }
+  </script>
+
+  <script src="//unpkg.com/alpinejs" defer></script>
+
+  <link rel="stylesheet" href="https://unpkg.com/@appwrite.io/pink" />
+  <link rel="stylesheet" href="https://unpkg.com/@appwrite.io/pink-icons" />
+</head>
+```
+
+And after the `</head>` tag we're going to add our `<body>` which will contain the actual form:
+
+```html
+<body>
+  <main class="main-content">
+    <div class="top-cover u-padding-block-end-56">
+      <div class="container">
+        <div class="u-flex u-gap-16 u-flex-justify-center u-margin-block-start-16">
+          <h1 class="heading-level-1">fal.ai Demo</h1>
+          <code class="u-un-break-text"></code>
+        </div>
+        <p class="body-text-1 u-normal u-margin-block-start-8" style="max-width: 50rem">
+          Use this page to test your implementation with FAL. Enter
+          text and receive the model output as a response.
+        </p>
+      </div>
+    </div>
+    <div class="container u-margin-block-start-negative-56"
+      x-data="{ prompt: '', result: { src: '' }, loading: false }">
+      <div class="card u-flex u-gap-24 u-flex-vertical">
+        <div class="u-flex u-cross-center u-gap-8">
+          <div class="input-text-wrapper is-with-end-button u-width-full-line">
+            <input x-model="prompt" type="search" placeholder="Prompt" />
+            <div class="icon-search" aria-hidden="true"></div>
+          </div>
+          <button class="button" x-bind:disabled="loading"
+            x-on:click="async () => { loading = true; result = { src: '' }; try { result = await onSubmit(prompt) } catch(err) { console.error(err); } finally { loading = false; } }">
+            <span class="text">Generate</span>
+          </button>
+        </div>
+        <template x-if="answer.type">
+          <div class="u-flex u-flex-vertical u-gap-12">
+            <div class="u-flex u-flex-vertical u-gap-12 card">
+              <div class="u-flex u-gap-12">
+                <h5 class="eyebrow-heading-2">Result:</h5>
+              </div>
+              <img class="u-max-width-400" x-bind:src="result.src" alt="FAL output" />
+            </div>
+          </div>
+        </template>
+      </div>
+    </div>
+  </main>
+</body>
+```
+
+All of this together will render a form that will submit your question to the Appwrite Function through a POST request which we'll create next. The Appwrite Function will call fal.ai's API and return the response, which will be displayed on your page using different conditional statements depending on the output media type.
+{% /section %}
+
+{% section #step-6 step=6 title="Handle POST Request" %}
+Now that we're serving a basic HTML page, we can add methods necessary to integrate with fal.ai's API.
+
+Import the fal.ai SDK at the top of our `main.js` file:
```

Comment:
```suggestion
Import the Fal.ai SDK at the top of our `main.js` file:
```
Do we want to use fai.ai or Fal.ai ? Once we decide the right approach we've to make sure we edit it across the file

Reply:
Going to stick with lowercase `fal.ai` as this is what they use